src/main/java/dev/waterdog/utils/ProxyConfig.java

```diff
@@ -64,7 +64,7 @@
 
     @Path("permissions")
     @Comment("Case-Sensitive permission list for players")
-    private Object2ObjectOpenHashMap<String, List<String>> playerPermissions = new Object2ObjectOpenHashMap<>() {{
+    private Object2ObjectOpenHashMap<String, List<String>> playerPermissions = new Object2ObjectOpenHashMap<String, List<String>>() {{
```

Comment:
This change was made so Java 8 knows which variable types is used in the inner class.